redhat-developer/developers.redhat.com
# [DEVELOPER-5401] Article base styles and layout

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5401

### Verification Process

1. Visit an article page
2. Verify that existing body is same width as mock
3. Verify published and updated info is below the article title and formatted correctly
4. Verify type matches mock styles
5. Verify on-page nav matches mocks and follows scroll

### Screenshots
![image](https://user-images.githubusercontent.com/10933938/46172817-746a8b80-c272-11e8-86b5-37133edf2642.png)
![image](https://user-images.githubusercontent.com/10933938/46172858-9532e100-c272-11e8-929d-7ec80ac6656e.png)
